q2_diversity/_ordination.py

```diff
@@ -9,5 +9,10 @@
 import skbio.stats.ordination
 
 
-def pcoa(distance_matrix: skbio.DistanceMatrix) -> skbio.OrdinationResults:
-    return skbio.stats.ordination.pcoa(distance_matrix)
+def pcoa(distance_matrix: skbio.DistanceMatrix, method: str = 'eigh',
+         number_of_dimensions: int = 0,
+         inplace: bool = False) -> skbio.OrdinationResults:
```

Comment:
Is `inplace` necessary at this time? From a qiime2 perspective, it's always parsing a file, in the future we might do something clever like skipping the format/disk if actions are pipelined together, but in that case we'd need to _not_ mutate at all.

Otherwise, the only time this would come up is if you were using the function directly, in which case, you're much better of using `skbio.stats.ordination.pcoa` which intends to be a library function.

Reply:
Yep, I think that's a good plan, let's leave inplace=False for QIIME 2 :+1: